src/api.ts

```diff
@@ -11,13 +11,15 @@ declare module 'koishi' {
 
 export class HitokotoApi extends Service {
   private _apiUrl: string
+  declare config: Config
```

Comment:
```suggestion
  config: Config
```

`declare` is redundant.